### ISSUE TITLE
Support for composite/batch

### ISSUE THREAD

**Issue by kvarga** [simple-salesforce/simple-salesforce#154]
I've got a local copy with support for composite/batch , but its rudimentary (fits my purposes, but probably not ready for everyone). 

Should I PR into a new branch on this repo? I'm saving significant time with my integrations by using this instead of upserting item-by-item. 

https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_composite_batch.htm

**Comment by pskrz** [comment]
@kvarga Could you share this code, please, I'm curious about how you designed it

**Comment by yeyeric** [comment]
@kvarga agreed with @ghost, could you please share you code for this ? Thanks

**Comment by kvarga** [comment]
Dang, I apologize for being "that guy" who posted something like this years ago but never shared.

The fork is pretty out-of-date (I left job where I was doing Python / SF Integration in May 2017), but found the code and pushed it to my fork: https://github.com/kvarga/simple-salesforce/commit/0dea0b38e4cc00541eeca42aca6009630f0fb0bc 

Sorry about that @ghost and @yeyeric . Hopefully this helps.
